src/presentation/http/router/note.ts

```diff
@@ -264,15 +263,68 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
   }, async (request, reply) => {
     const noteId = request.note?.id as number;
     const content = request.body.content as JSON;
-    const parentId = request.body.parentId;
 
-    const note = await noteService.updateNoteContentById(noteId, content, parentId);
+    const note = await noteService.updateNoteContentById(noteId, content);
 
     return reply.send({
       updatedAt: note.updatedAt,
     });
   });
 
+  /**
+   * Update note relation by id.
+   */
+  fastify.patch<{
+    Params: {
+      notePublicId: NotePublicId,
+    },
+    Body: {
+      parentId: NotePublicId,
+    },
+    Reply: {
+      isUpdated: boolean,
+    }
+  }>('/:notePublicId/relation', {
+    schema: {
+      params: {
+        notePublicId: {
```

Comment:
client should not use the "publicId" term. It uses just "id", so lets rename it to the `noteId` and `parentNoteId`

Reply:
I cant rename notePublicIdId because I will need to change note resolver and change a lot of routes. We can do it in another PR

Reply:
yes, sorry, there is no problem with `notePublicId`, since it is just a name from path: `POST /note/:notePublicId/relation`, so client is not using the `parent` word.